WebRoot/js/zimbraMail/share/model/ZmAutocomplete.js

```diff
@@ -549,8 +549,7 @@ ZmAutocompleteMatch = function(match, options, isContact, str) {
  */
 ZmAutocompleteMatch.prototype.setContactGroupMembers =
 		function(groupId, callback) {
-			var ac = window.parentAppCtxt || window.appCtxt;
-			var contactGroup = ac.cacheGet(groupId);
+			var contactGroup = appCtxt.getById(groupId);
```

Comment:
@sumitTelus issue is still persist with latest build on apps-bravo2 (provided by you).
https://synacor.atlassian.net/browse/ZBUG-4874?focusedCommentId=348773